magit-todos.el

```diff
@@ -397,15 +397,16 @@ Only necessary when option `magit-todos-update' is nil."
   "Show current item.
 If PEEK is non-nil, keep focus in status buffer window."
   (interactive)
-  (let* ((status-window (selected-window))
-         (item (oref (magit-current-section) value))
-         (buffer (magit-todos--item-buffer item)))
-    (pop-to-buffer buffer)
-    (magit-todos--goto-item item)
-    (when (derived-mode-p 'org-mode)
-      (org-show-entry))
-    (when peek
-      (select-window status-window))))
+  (if-let* ((status-window (selected-window))
+            (item (oref (magit-current-section) value))
+            (is-valid-item (cl-struct-p item))
+            (buffer (magit-todos--item-buffer item)))
+      (progn (pop-to-buffer buffer)
+             (magit-todos--goto-item item)
+             (when (derived-mode-p 'org-mode)
+               (org-show-entry))
+             (when peek
+               (select-window status-window)))))
```

Comment:
There should probably be a `message` in the `else` form that says something like, `Not a todo item`.